app.js

```diff
@@ -72,16 +83,11 @@ function getRandomInt(max) {
 
 function encryptSentence(inputSentence) {
   let encryptedArray = [...inputSentence].map(letter => {
-    if (letter === ' ') {
-      encryptedLetter = '-';
-    } else {
-      encryptedLetter = '_';
-    }
-    return encryptedLetter;
+    return (letter === ' ') ? (encryptedLetter = '-') : (encryptedLetter = '_');
   })
   return encryptedArray.join('');
 }
 
-letterCheck() {
-  // to be continued...
+function letterCheck(clickedLetter) {
+  alert(clickedLetter);
```

Comment:
my advice is to not use alert. It is not comfortable. Use `console.log`, but it is not important in this PR. You don't need to change it